asmcomp/CSEgen.ml

```diff
@@ -237,11 +237,12 @@ method class_of_operation op =
   | Ispecific _ -> Op_other
   | Iname_for_debugger _ -> Op_pure
   | Ipoll -> Op_checkbound
-  | Inop -> Op_store true (* not correct, but enough to force CSE from leaving
-                             nops alone *)
+  | Inop ->
+      (* not correct, but enough to force CSE from leaving nops alone *)
+      Op_store true
+  | Idls_get -> Op_pure
```

Comment:
I'm not a deep expert, but is it correct that:
```ocaml
| Iload(_,_) -> Op_load
```
but 
```ocaml
| Idls_get -> Op_pure
```
I'm not sure I understand why `Idls_get` shouldn't be the same as a `Iload`

Reply:
I shall change this one to `Iload`.